Mathlib/RingTheory/Etale/Field.lean

```diff
@@ -17,6 +17,11 @@ Let `K` be a field, `A` be a `K`-algebra and `L` be a field extension of `K`.
     If `L` is separable over `K`, then `L` is formally etale over `K`.
 - `Algebra.FormallyEtale.iff_isSeparable`:
     If `L` is (essentially) of finite type over `K`, then `L/K` is etale iff `L/K` is separable.
+- `Algebra.FormallyEtale.iff_exists_algEquiv_prod`:
+    If `A` is (essentially) of finite type over `K`,
+    then `A/K` is etale iff `A` is a finite product of separable field extensions.
```

Comment:
I think in the text we should include the accent and write "étale", as is the case with people's names etc.